<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WriteMessageDialog</class>
 <widget class="QDialog" name="WriteMessageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Write Message</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labelName">
         <property name="text">
          <string>&amp;Name:</string>
         </property>
         <property name="buddy">
          <cstring>lineEditName</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditName"/>
       </item>
       <item>
        <widget class="QLabel" name="labelMailAddress">
         <property name="text">
          <string>&amp;Mail Address:</string>
         </property>
         <property name="buddy">
          <cstring>lineEditMailAddress</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditMailAddress"/>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxSage">
         <property name="text">
          <string>&amp;Sage</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="textEditMessage"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
